pom.xml

```diff
@@ -35,10 +35,15 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>javax.inject</groupId>
+				<artifactId>javax.inject</artifactId>
+				<version>1</version>
+			</dependency>
```

Comment:
Deze ben ik niet helemaal zeker over of ik dit op een andere manier had moeten doen

Reply:
Deze komt binnen via `org.apache.maven:maven-core` en de fout is dat die dan niet gemanaged wordt, dus volgensmij moet die dan gewoon geëxclude worden in de onderwijs parent.

Reply:
Ja, deze heb je wel echt nodig. Hij is normaal banned (en daarom ook niet gemanaged), maar dat is hier niet van toepassing, want dit is geen JEE applicatie.